crates/rpc/src/v02/method/get_transaction_receipt.rs

```diff
@@ -327,6 +328,12 @@ mod types {
     #[cfg_attr(any(test, feature = "rpc-full-serde"), derive(serde::Deserialize))]
     #[serde(deny_unknown_fields)]
     pub struct MessageToL1 {
+        // In RPC spec v0.3 MSG_TO_L1 has mandatory `from_address` field.
+        // This way it can be supported for both v0.2 and v0.3 versions.
+        #[serde_as(as = "Option<RpcFelt>")]
+        #[serde(default)]
+        #[serde(skip_serializing_if = "Option::is_none")]
+        pub from_address: Option<Felt>,
```

Comment:
Can this one be a `ContractAddress`?

Reply:
The [spec](https://github.com/starkware-libs/starknet-specs/blob/v0.3.0/api/starknet_api_openrpc.json#L2288) defines it as a `Felt`, but OK.